tests/integration/conftest.py

```diff
@@ -122,8 +122,8 @@ def _generate_test_files(
 
 # test helper specific to Domains test suite
 @pytest.fixture
-def create_master_domain():
-    timestamp = str(int(time.time()))
+def master_domain():
+    timestamp = str(int(time.time_ns()))
```

Comment:
`time.time_ns()` is already an integer, so we don't have to cast it.
```suggestion
    timestamp = str(time.time_ns())
```